src/style.css

```
@@ -1,43 +1,16 @@
 @import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap');
 
 :root {
-    --bgcolor: #785D9B;
     --bgcolor-help: #ffbdcf;
     --txtcolor: #FFFEF4;
     --contenttxtcolor: #000408;
```

```suggestion
    --helptxtcolor: #000408;
```
Cambierei quei e dove è usata, cioè in .listHelp a.
Così diamo un nome adatto e dove viene utilizzato.